src/partitioner/stream/Partitioner.cpp

```diff
@@ -209,8 +210,8 @@ partitionedEdge Partitioner::fennelPartitioning(std::pair<int, int> edge) {
  * Expect a space seperated pair of vertexts representing an edge in the graph.
  **/
 std::pair<long, long> Partitioner::deserialize(std::string data) {
-    std::vector<std::string> v = JasmineGraphIncrementalStore::_split(data, ' ');
-    // std::cout << "Vertext 1 = " << stoi(v[0]) << std::endl;
-    // std::cout << "Vertext 2 = " << stoi(v[1]) << std::endl;
+    std::vector<std::string> v = Partition::_split(data, ' ');
+    std::cout << "Vertext 1 = " << stoi(v[0]) << std::endl;
```

Comment:
Why do we need these couts?

Reply:
These are for debugging purpose, Hence converted them to debug logs
Fixed in https://github.com/miyurud/jasminegraph/pull/122/commits/bdd734e88390169ddc9e44c74030d1c1b170d0ae

Reply:
Ok, Done